File: C++/Maths/Calculate the sum of natural numbers by formula and conditional check.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
    cout << "Enter a postive number: ";
    cin >> n;
    if( n >= 0){
        int sum = n*(n+1)/2;
        cout <<  "The sum of nth numbers is = " << sum;
    }else{
        cout << "Entered number isn't a positive number";
    }
}

File: Java/Maths/Calculate the sum of natural numbers by formula and conditional check.java
import java.util.Scanner;
class sumOfn_conditional {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Enter a postive number: ");
        int n = input.nextInt();
        if( n >= 0){
            int sum = n*(n+1)/2;
            System.out.println( "The sum of nth numbers is = " + sum );
        }else{
            System.out.println("Entered number isn't a positive number");
        }
    }
}